packages/node_modules/@node-red/editor-client/src/sass/tab-config.scss

```
@@ -113,6 +113,9 @@ ul.red-ui-sidebar-node-config-list li.red-ui-palette-node-config-type {
         margin-right: 5px;
     }
 }
+.red-ui-palette-node-config-invalid {
+    border-color: var(--red-ui-node-border-unknown)
```

```suggestion
    border-color: var(--red-ui-form-input-border-error-color)
```

To make it consistent with the color used for the input field borders when they have errors.

---

@bonanitech I did consider that, just wasn't sure about mixing the form styling with the node styling. And wanted to avoid adding yet another variable to minimise the impact on custom themes.

I guess we can go with it for now - as I mentioned in the PR, I may yet refine this to use the 'triangle' badge we use in the main workspace.

---

This is a good thing 🙂, the addition of the triangle will indeed be very useful 😋 
We should also talk about this [thread](https://discourse.nodered.org/t/fr-validationerrors-for-config-node-credentials/83105) which will have an impact on this PR.